mage/main.go

```diff
@@ -40,6 +40,8 @@ const mainfile = "mage_output_file.go"
 const initFile = "magefile.go"
 
 var (
+	force, verbose, list, help, mageInit, keep, showVersion bool
```

Comment:
this can get removed.... with switch to a standalone parse function, we don't have global flag variables anymore.